search/hibernate-search-lucene/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.hibernate.testcasetemplate</groupId>
+	<artifactId>test-case-template-hibernate-search-lucene</artifactId>
+	<version>1.0.0.Final</version>
+	<name>Hibernate Search with Lucene Test Case Template</name>
+
+	<properties>
+		<version.org.hibernate.search>5.6.0.Beta1</version.org.hibernate.search>
```

Comment:
The stable will be what most people use (besides adventurous ES explorers),
so I'd stick to that for Lucene.